public/api/conf/1.0/schemas/state-pension.json

```diff
@@ -0,0 +1,180 @@
+            "description": "The current date is equal to or greater than the day before the customer&#8217;s State Pension date. The customer is eligible for State Pension and therefore we do not provide a forecast."
+          }
+        ],
+        "description": "A list of exclusion reasons",
+        "example": "Abroad"
+        }
+    },
+    "earningsIncludedUpTo": {
+      "type": "string",
+      "description": "The date which the calculations are based on. This is always the end date of the last posted tax year",
+      "example": "2015-04-05"
+    },
+    "amounts": {
+      "type": "object",
+      "properties": {
+        "protectedPayment": {
+          "type": "boolean",
+          "description": "Whether the customer has an amount higher than the full rate at the end of the 2015/16 tax year",
+          "example": "false"
+        },
+        "current": {
+          "type": "object",
+          "properties": {
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per week",
+              "example": "133.41"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per month",
+              "example": "580.1"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The amount of accrued State Pension to date per year",
+              "example": "6961.14"
+            }
+          }
+        },
+        "forecast": {
+          "type": "object",
+          "properties": {
+            "yearsToWork": {
+              "type": "number",
+              "description": "The number of future qualifying years needed to achieve the forecast amount",
+              "example": "1"
+            },
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per week",
+              "example": "146.76"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per month",
+              "example": "638.14.1"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per year",
+              "example": "7657.73"
+            }
+          }
+        },
+        "maximum": {
+          "type": "object",
+          "properties": {
+            "yearsToWork": {
+              "type": "number",
+              "description": "The number of future qualifying years needed to achieve the maximum amount",
+              "example": "3"
+            },
+            "gapsToFill": {
+              "type": "number",
+              "description": "The minimum number of National Insurance gaps that need to be filled to achieve the maximum amount",
```

Comment:
The minimum number of National Insurance gaps that need to be filled to get the most possible